includes/class-amp-theme-support.php

```diff
@@ -2463,6 +2470,59 @@ public static function enqueue_assets() {
 		wp_styles()->add_data( 'amp-default', 'rtl', 'replace' );
 	}
 
+	/**
+	 * Enqueue AMP paired browsing client script.
+	 *
+	 * @return void
+	 */
+	public static function enqueue_paired_browsing_client() {
+		wp_enqueue_script(
```

Comment:
For the version and the list of dependencies we can use something like this:

```php
$asset_file   = AMP__DIR__ . '/assets/js/amp-paired-browsing-client.asset.php';
$asset        = require $asset_file;
$dependencies = $asset['dependencies'];
$version      = $asset['version'];

wp_enqueue_script(
	'amp-paired-browsing-client',
	amp_get_asset_url( '/js/amp-paired-browsing-client.js' ),
	$dependencies,
	$version,
	true
);
```

This way we get the list of dependencies automatically for free, and the script version changes every time there is an actual change to the script (no need to clear caches).